app/controllers/api/v2/node/containers_controller.rb

```diff
@@ -12,6 +12,17 @@ def scheduled
     render json: ::Api::V2::Node::ContainerSerializer.new(@containers).to_h
   end
 
+  # GET /bootstrap_scheduled
+  # Get all bootstrap scheduled containers within a particular node
+  def bootstrap_scheduled
+    @containers = current_node.containers.
+      where(status: [
+        ::Container.statuses[:provisioned],
+        ::Container.statuses[:bootstrap_error],
```

Comment:
does this line means if a container bootstrap process is error before, it will retry and retry again?

Reply:
With this commit, it is. 

Modified the behaviour to exclude from auto retrying with this new commit link https://github.com/pathfinder-cm/pathfinder-mono/pull/22/commits/86af73769eeaffd9ec77f00b65f29162f68cc033